big_tests/tests/fast_auth_token_SUITE.erl

```diff
@@ -510,3 +714,45 @@ initial_response_elem(Payload) ->
 spec_to_lus(Spec) ->
     #{username := Username, server := Server} = maps:from_list(Spec),
     jid:to_lus(jid:make_bare(Username, Server)).
+
+escalus_client_to_pid(#client{rcv_pid = Pid}) ->
+    Pid.
+
+escalus_client_to_props(#client{props = Props}) ->
+    Props.
+
+set_ticket(Data = #{spec := Spec}, Ticket) ->
+    Map = proplists:to_map(Spec),
+    #{ssl_opts := SslOpts} = Map,
+    Map2 = Map#{
+         %% Set Ticket to be used with the next reconnect
+         ssl_opts => [{use_ticket, [Ticket]} | SslOpts]},
+    Data#{spec => proplists:from_map(Map2)}.
+
+maybe_receive_session_tickets_on_connect(Client, Data = #{spec := Spec}) ->
+    case proplists:get_value(receive_session_tickets_on_connect, Spec, false) of
+        true ->
+            Ticket = receive_ticket(Client),
+            set_ticket(Data, Ticket);
+        false ->
+            Data
+    end.
+
+add_tls_opts(C2SListener = #{tls := TLS}, Extra) ->
+    C2SListener#{tls => maps:merge(TLS, Extra)}.
+
+%% Reads DB using RPC.
+%% Allows to write better tests, even if it breaks "only XMPP access approach".
+assert_current_token_count(Config, Spec, ExpectedCount, Text) ->
+    HostType = domain_helper:host_type(),
+    {LUser, LServer} = spec_to_lus(Spec),
+    AgentId = user_agent_id(),
+    Args = [HostType, LServer, LUser, AgentId],
+    {ok, Data} = distributed_helper:rpc(distributed_helper:mim(), mod_fast_auth_token_backend, read_tokens, Args),
+    ?assertMatch(#{current_count := ExpectedCount}, Data, #{expected_count => ExpectedCount, text => Text}).
+
+db_engine() ->
+    escalus_ejabberd:rpc(mongoose_rdbms, db_engine, [domain_helper:host_type()]).
```

Comment:
We've been trying to make escalus less ejabberd/mongooseim specific and use the rpc helpers we have in `distributed_helper`, why this is back?

Reply:
mssql fails with timeout when we execute all tests in parallel.
I can move the check into distributed_helper.